pkg/server/server.go

```diff
@@ -1928,6 +1928,7 @@ func serveUIAssets(fileServer http.Handler, cfg Config) http.Handler {
 		tmplArgs := ui.IndexHTMLArgs{
 			ExperimentalUseLogin: cfg.EnableWebSessionAuthentication,
 			LoginEnabled:         cfg.RequireWebSession(),
+			Tag:                  build.GetInfo().Tag,
```

Comment:
looks like this is what broke the `ServeIndexHTML` test in CI. It's pretty brittle — asserts the exact HTML that comes out. Not sure how to make that always pass, since the build tag will keep changing… Maybe can just sprintf it into the expected HTML.

Reply:
Tests updated.